docs/ref/django-admin.txt

```diff
@@ -656,6 +532,13 @@ For example::
 
     django-admin dumpdata --format=json --database=test app_label.ModelName | django-admin loaddata --format=json --database=prod -
 
+The :djadmin:`dumpdata` command can be used to generate input for ``loaddata``.
+
+.. seealso::
+
+    For more detail about how fixtures are made, used, found and saved see the
```

Comment:
```suggestion
    For more detail about how fixtures are made, used, found, and saved see the
```
Comma is missing before 'or'/'and' (see https://en.wikipedia.org/wiki/Serial_comma).